firmware/include/model/weatherDataModel.h

```diff
@@ -41,6 +41,9 @@ class WeatherDataModel {
     float getDayLow(int num);
     String getDayLow(int8_t num, int8_t digits);
 
+    WeatherDataModel &setLastUpdate(String lastupdate);
```

Comment:
Why does this return itself?

Reply:
Sorry, i don't understand the question. What's the problem here ? I copied the code from the other defintions and only replaced the variable.

Reply:
It's the same code as here, only with setlastupdate and lastupdate as variable:

WeatherDataModel &setCityName(String city);
String getCityName();